src/templates/pt/2019/base.html

```diff
@@ -0,0 +1,229 @@
+Web Almanac é um relatório abrangente sobre o estado da Web, apoiado por dados reais e especialistas confiáveis da Web.
+É composto por 20 capítulos que abrangem aspectos do conteúdo da página, experiência do usuário, publicação e
+distribuição.
+{% endblock %}
+
+{% block http_archive_link %}Página inicial HTTP Archive{% endblock %}
+
+{% block page_navigation %}Navegação de página{% endblock %}
+
+{% block help_translate %}Ajudar Traduzir{% endblock %}
+{% block translation_not_available %}Inglês - não disponível em Português{% endblock %}
+{% block language_switcher %}Selecione o Idioma{% endblock %}
+{% block year_switcher %}Selecione o Ano{% endblock %}
+
+{% block home %}Início{% endblock %}
+{% block table_of_contents_title %}Tabela de Conteúdos{% endblock %}
+{% block contributors_title %}Contribuintes{% endblock %}
+{% block methodology_title %}Metodologia{% endblock %}
+
+{% block part %}Parte{% endblock %}
+{% block chapter %}Capítulo{% endblock %}
+
+{% block start_exploring %}Comece a explorar{% endblock %}
+
+{% block menu_title %}Menu da Página{% endblock %}
+{% block open_the_menu %}Abra o menu{% endblock %}
+{% block close_the_menu %}Feche o menu{% endblock %}
+
+{% block index_nav_title %}Chapter table of contents{% endblock %}
+{% block index_title %}Index{% endblock %}
+{% block open_the_index %}Abrir índice de conteúdos{% endblock %}
+{% block close_the_index %}Fechar índice de conteúdos{% endblock %}
+
+{% block and %}e{% endblock %}
+
+{% block open %}Abrir{% endblock %}
+{% block close %}Fechar{% endblock %}
+
+{% macro show_description(id) %}Mostrar descrição da figura {{ id }}{% endmacro %}
+{% macro hide_description(id) %}Ocultar descrição da figura {{ id }}{% endmacro %}
+
+{% block written_by_before %}Escrito por{% endblock %}
+{% block reviewed_by_before %}Revisado por{% endblock %}
+{% block translated_by_before %}Traduzido por{% endblock %}
+{% block written_by_after %}{% endblock %}
+{% block reviewed_by_after %}{% endblock %}
+{% block translated_by_after %}{% endblock %}
+
+{% block author %}Autor(a){% endblock %}
+{% block authors %}Autores{% endblock %}
+
+{% macro onTwitter(twitterHandle) %}@{{twitterHandle}} no Twitter{% endmacro %}
+{% macro onGitHub(gitHubHandle) %}{{gitHubHandle}} no GitHub{% endmacro %}
+{% macro website(authorName) %}{{authorName}} website{% endmacro %}
+
+{% macro edition(year) %}{{ year }} Edição{% endmacro %}
+
+{% block footer_title %}Navegação no rodapé{% endblock %}
+{% block copyright %}© Web Almanac. Licenciado sob <a
+  href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/LICENSE">Apache 2.0</a>.{% endblock %}
+{% block accessibility_statement %}Declaração de acessibilidade{% endblock %}
+
+{% block contributors_description %}
+O Web Almanac Isso foi possível graças ao trabalho árduo da comunidade da web. {{ self.contributors() }} as pessoas
+ofereceram inúmeras horas nas fases de planejamento, pesquisa, redação e produção.
```

Comment:
Combine these to keep line numbering same as English:

```suggestion
O Web Almanac Isso foi possível graças ao trabalho árduo da comunidade da web. {{ self.contributors() }} as pessoas ofereceram inúmeras horas nas fases de planejamento, pesquisa, redação e produção.
```